scripts/cnode-helper-scripts/guild-deploy.sh

```diff
@@ -3,6 +3,7 @@
 # shellcheck source=/dev/null
 
 unset CNODE_HOME
+unset MITHRIL_HOME
```

Comment:
I am not sure if mithril should need a seperate top level folder, and even if there was a strong reason to do so - it would have required contention across scripts to be able to set the path via flags, but really - I'd rather this path to be set as a subset of existing top level folder

Reply:
Their documentation starts with `/opt/mithril` as their source for storage. In theory Mithril is not limited to Cardano as it will support other blockchains and potentially cardano sidechains. 

If `guild-deploy.sh` will not be used in the future to deploy a sidechain of cardano then having storage in `/opt/cardano/cnode/mithril-signer` instead makes perfect sense, as well as nativaly allows a mithril signer per VNODE instance without extending VNODE into `/opt/mithril/mithril-signer` etc.

As long as this is being considered while choosing, I am open to whatever path the guild prefers to use.